runtime/src/invoke_dpe.rs

```diff
@@ -0,0 +1,42 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{
+    CptraDpeTypes, DpeCrypto, DpeEnv, DpePlatform, Drivers, InvokeDpeReq, InvokeDpeResp,
+    MailboxResp, MailboxRespHeader,
+};
+use caliptra_drivers::{CaliptraError, CaliptraResult};
+use zerocopy::FromBytes;
+
+const INVOKE_DPE_LOCALITY: u32 = 0x30020004;
+
+pub struct InvokeDpeCmd;
+impl InvokeDpeCmd {
+    pub(crate) fn execute(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<MailboxResp> {
+        if let Some(cmd) = InvokeDpeReq::read_from(cmd_args) {
+            // Won't panic, full_digest is always larger than digest
+            let mut response_buf = [0u8; InvokeDpeResp::DATA_MAX_SIZE];
+            let mut env = DpeEnv::<CptraDpeTypes> {
+                crypto: DpeCrypto::new(&mut drivers.sha384),
+                platform: DpePlatform,
+            };
+            match drivers
+                .dpe
+                .execute_serialized_command(&mut env, INVOKE_DPE_LOCALITY, &cmd.data)
```

Comment:
I think we should set the locality to the mailbox pauser: https://chipsalliance.github.io/caliptra-rtl/main/internal-regs/?p=clp.mbox_csr.mbox_user. I think you'll need to add support to the mailbox driver to read this register.

Reply:
That is the address of that register. The SoC will write to that address to set the caller whenever a mailbox command is sent. So on each mailbox command we'll need to read from that register to get the value. You can use look at the [mailbox driver](https://github.com/chipsalliance/caliptra-sw/blob/7e9dc76e9083c09027c476d1776f1b96b3caea72/runtime/src/mailbox.rs#L24) to see how we read from other registers.